projects/fsastorefrontlib/src/shared/util/helpers/PaginationHelper.ts

```diff
@@ -0,0 +1,33 @@
+import { PaginationModel } from '@spartacus/core';
```

Comment:
Seems we have a mix of things in this folder. I would suggest to perhaps split these helpers into some groups or move this one to b2b/utility or something like that?